samples/cpp/tutorial_code/ImgProc/Pyramids.cpp

```diff
@@ -39,15 +39,14 @@ int main( void )
   dst = tmp;
 
   //![create_window]
-  namedWindow( window_name, WINDOW_AUTOSIZE );
   imshow( window_name, dst );
   //![create_window]
 
   //![infinite_loop]
   for(;;)
   {
     int c;
-    c = waitKey(10);
+    c = waitKey(0);
```

Comment:
Oh BTW, take a look [here](https://github.com/opencv/opencv/pull/7105). It is important not to forget the anding. 

Thus this line should be

`c = waitKey(0) & 0xFF;`